### PR TITLE
[EA] Merge master

### PR DESCRIPTION
**Pull request by LukeShu**
This PR merges `master` (actually, the branch that is https://github.com/datawire/ambassador-docs/pull/168 against master) in to `early-access`, so that

 1. The EA docs get fixes that have been made in the GA docs, and
 2. The EA docs get website previews on PRs

### CODE REVIEW

File: reference/core/ambassador.md [hunk 1]
```diff
@@ -177,11 +177,13 @@ If set, `cluster_idle_timeout_ms` specifies the timeout (in milliseconds) after
 
 ### gRPC HTTP/1.1 bridge (`enable_grpc_http11_bridge`)
 
-Ambassador Edge Stack supports bridging HTTP/1.1 clients to backend gRPC servers. When an HTTP/1.1 connection is opened and the request content type is `application/grpc`, Ambassador Edge Stack will buffer the response and translate into gRPC requests. For more details on the translation process, see the [Envoy gRPC HTTP/1.1 bridge documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_http1_bridge_filter). This setting can be enabled by setting `enable_grpc_http11_bridge: true`.
+Ambassador Edge Stack supports bridging HTTP/1.1 clients to backend gRPC servers. When an HTTP/1.1 connection is opened and the request content type is `application/grpc`, Ambassador Edge Stack will buffer the response and translate into gRPC requests. For more details on the translation process, see the [Envoy gRPC HTTP/1.1 bridge documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_http1_bridge_filter). This setting can be enabled by setting `enable_grpc_http11_bridge: true`. Read more about [gRPC and Ambassador Edge Stack](user-guide/grpc).
```
Comment (scoyle391) [inline reference/core/ambassador.md]: will the link to the user-guide/grpc work as is now? aka, we aren't using the ../.. anymore?

File: reference/core/ingress-controller.md [hunk 2]
```diff
@@ -1,72 +1,18 @@
 # Ambassador as an Ingress Controller
 
-Starting with version 0.80.0, the Ambassador Edge Stack can act as a Kubernetes [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/), reading configuration data from Kubernetes [`Ingress`](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources. This makes it easier to work with other `Ingress`-oriented tools within the Kubernetes ecosystem, and it makes it easier for users migrating from other ingress controllers to try the Ambassador Edge Stack.
+An `Ingress` resource is a popular way to expose Kubernetes services to the Internet. In order to use `Ingress` resources, you need to install an [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/). The Ambassador Edge Stack can function as a full-fledged Ingress controller, making it easy to work with other `Ingress`-oriented tools within the Kubernetes ecosystem.
```
Comment (scoyle391) [inline reference/core/ingress-controller.md]: fully-fledged

File: about/quickstart.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ In this Docker quickstart guide, we'll get Ambassador Edge Stack running locally
 
 By default, Ambassador Edge Stack uses a demo configuration to show some of its basic features. Get it running with Docker, and expose Ambassador Edge Stack on port 8080:
 
-```shell
+```
```
Comment (kflynn) [inline about/quickstart.md]: Seems like we probably shouldn't change this?

File: reference/core/ingress-controller.md (hunk 2 above)
Comment (kflynn) [reply reference/core/ingress-controller.md]: Hmmmm. I've always heard "full-fledged" (in the technical world anyway).

File: user-guide/grpc.md [hunk 4]
```diff
@@ -199,7 +199,7 @@ spec:
   hosts:
   - "*"
   secret: ambassador-cert
-  alpn_protocol: h2
+  alpn_protocols: h2
```
Comment (kflynn) [inline user-guide/grpc.md]: It disturbs me that I had to go look this up. :wink: (the plural is indeed correct)

**Comment by LukeShu** [comment]
If it's all-right, I'll handle those in a follow-up PR.

File: reference/core/ambassador.md (hunk 1 above)
Comment (LukeShu) [reply reference/core/ambassador.md]: You're right, and it's broken in the currently-live version of the GA docs too.  I guess I'll do some work on the broken link checker.

File: reference/core/ingress-controller.md (hunk 2 above)
Comment (LukeShu) [reply reference/core/ingress-controller.md]: Well, "fledge" is a verb, and "full[y]" is modifying that verb, making it an adverb not an adjective, and the adverbs usually end in "ly".

File: about/quickstart.md (hunk 3 above)
Comment (LukeShu) [reply about/quickstart.md]: 8bf2b5d217e08e5c67644a0be1a692e8246c53bb

It seems that the shell syntax-highlighting was interfering with `$variable$ substitution :/

File: reference/core/ambassador.md (hunk 1 above)
Comment (LukeShu) [reply reference/core/ambassador.md]: I've fixed this separately in https://github.com/datawire/ambassador-docs/pull/172 (EA) and https://github.com/datawire/ambassador-docs/pull/173 (GA)

File: reference/core/ingress-controller.md (hunk 2 above)
Comment (LukeShu) [reply reference/core/ingress-controller.md]: Fixed here for EA, and in https://github.com/datawire/ambassador-docs/pull/186 for GA.

**Pull request merged**